app/(auth)/login/AuthForm.tsx

```diff
@@ -15,67 +15,71 @@ export default function AuthForm() {
     try {
       const response = await axios.post('api/auth/login', formData);
       if (response.status === HttpStatusCode.Ok) router.push('/users');
+      else
+        throw new AxiosError(
+          undefined,
+          response.status as unknown as string,
+          response.config,
+          response.request,
+          response
+        );
     } catch (err) {
       const error = err as AxiosError;
-      Swal.fire({
-        icon:               'error',
-        title:              'Oops...',
-        html:               `Hubo un error.<pre>Error: ${error.response?.status} - ${error.response?.statusText}</pre>`,
-        background:         'rgb(24 24 27)',
-        color:              '#FFFFFF',
-        confirmButtonColor: '#3085d6',
-      });
+      setLoginError(
+        `${error.response?.status} - ${error.response?.statusText}`
+      );
+      document.getElementById('error-modal')?.click();
+      setIsLoading(false);
     }
-    setIsLoading(false);
   };
 
   return (
-    <form
-      onSubmit={handleSubmit(data => onSubmit(data))}
-      className='grid grid-cols-1 gap-4 w-11/12 max-w-md p-5 rounded bg-white dark:bg-zinc-900 shadow'
-    >
-      <TextField
-        label='Correo electrónico'
-        type='text'
-        {...register('email', {
-          ...validateEmail(true),
-        })}
-        error={errors.email ? true : false}
-        helperText={errors.email ? (errors.email.message as string) : ' '}
-      />
-      <TextField
-        label='Contraseña'
-        type='password'
-        {...register('password', {
-          required: 'Este campo es requerido',
-        })}
-        error={errors.password ? true : false}
-        helperText={errors.password ? (errors.password.message as string) : ' '}
-      />
-      <Button
-        type='submit'
-        size='large'
-        variant='contained'
-        disabled={isLoading}
+    <>
+      <form
+        onSubmit={handleSubmit(onSubmit)}
+        className='grid grid-cols-1 gap-4 w-11/12 max-w-md p-5 border-base-content/5 border bg-base-200 rounded-box'
       >
-        {isLoading ? (
-          <span className='flex justify-center items-center'>
-            <CircularProgress color='inherit' size='1.5rem' className='mr-2' /> Cargando...
-          </span>
-        ) : (
-          'Iniciar sesión'
-        )}
-      </Button>
-      <Button size='large' LinkComponent={Link} href='forgot-password'>
-        Recuperar contraseña
-      </Button>
-    </form>
+        <TextInput
+          name='email'
+          label='Correo electrónico'
+          type='email'
+          register={register}
+          errorMessage={errors.email?.message as string}
+          validation={validateEmail(true)}
+        />
+        <TextInput
+          name='password'
+          label='Contraseña'
+          type='password'
+          register={register}
+          errorMessage={errors.password?.message as string}
+          required
+        />
+        <button
```

Comment:
Component "button" is all lowercase. Same thing with "form". Is that on purpose?